com.unity.netcode.gameobjects/Tests/Runtime/BaseMultiInstanceTest.cs

```diff
@@ -180,6 +180,11 @@ private bool ClientSceneHandler_CanClientsLoad()
             return CanClientsLoad();
         }
 
+        protected virtual void OnBeforeClientStart()
```

Comment:
Would there be a way to not need an empty function? Passing null as a delegate or having a boolean? It is a bit puzzling to need a new function that does nothing.

Reply:
+1, I think `() => {}` would just work

Reply:
When I say ”what was there before”, I think there's a commit I made to experimental-develop that had it that way, and it eas a maintainability issue because `StartSomeClientsAndServerWithPlayers` is called by `Setup`. Personally I want to move `updatePlayerPrefab` to be a base class function for the same reason but it's not in the scope of this PR. Any implementation that requires copying and pasting code from the base class to a child class is dangerous.